src/nylas.js

```diff
@@ -29,16 +30,15 @@ class Nylas {
       this.apiServer = apiServer;
     }
 
-    let conn;
+    const conn = new NylasConnection(this.appSecret, { clientId: this.appId });
+    this.webhooks = new ManagementModelCollection(Webhook, conn, this.appId);
```

Comment:
oooh smart - i like how you made it accessible on the Nylas model so ppl don't have to make their own instance of an account management model